server/src/main/java/com/cloud/server/StatsCollector.java

```diff
@@ -1209,7 +1212,7 @@ private String getAutoscaleAction(HashMap<Long, Double> avgCounter, long groupId
             for (AutoScaleVmGroupPolicyMapVO asVmgPmap : listMap) {
                 AutoScalePolicyVO policyVO = _asPolicyDao.findById(asVmgPmap.getPolicyId());
                 if (policyVO != null) {
-                    Integer quitetime = policyVO.getQuietTime();
```

Comment:
The variable 'quitetime' is only assigned values of primitive type and is never 'null', but it is declared with the boxed type 'Integer'.